src/main/java/it/gov/pagopa/pu/debtpositions/util/Utilities.java

```diff
@@ -0,0 +1,15 @@
+package it.gov.pagopa.pu.debtpositions.util;
+
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+
+public class Utilities {
+
+  private Utilities() {
+  }
+
+  public static OffsetDateTime localDatetimeToOffsetDateTime(LocalDateTime localDateTime) {
+    return localDateTime.atOffset(ZoneOffset.UTC);
```

Comment:
localDateTime parameter could be null (createDate and updateDate are filled during the save operation)